docs/_docs/Instructors/01-Teaching-process.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Teaching Process
+category: Instructors
+order: 1
+---
+
+Contents:
+- [Material structure](#materials)
```

Comment:
Not sure if this is still WIP, but as of now the links to section given don't seem to agree with the section titles.

Reply:
Should be fixed now